src/client/scripts/components/Navbar1.js

```diff
@@ -12,29 +12,31 @@ export class Navbar1 {
     }
     
     async render() {
-        const elm = document.createElement('nav');
+        const elm = document.createElement('div');
         elm.classList.add('navbar1');
 
         elm.innerHTML = `
         <nav>
-            <a href="#landing" id="nav-landing">Home</a>
-            <a href="#about" id="nav-about">About us</a>
-            <a href="#sign-in" id="nav-sign-in">Sign in</a>
+            <div class="main-links">
+                <a href="#landing" id="nav-landing">Home</a>
+                <a href="#about" id="nav-about">About us</a>
+                <a href="#sign-in" id="nav-sign-in">Sign in</a>
+            </div>
         </nav>
+        <hr>
         `;
-        // TODO: style, etc.
 
-        // add click event listener to each link
         elm
             .querySelectorAll('a')
-            .forEach((link) =>
```

Comment:
Undo your changes for lines 30-36, the code I wrote is specific to Navbar1

Reply:
Original code is in the main branch:
https://github.com/ashleybhandari/team-friendship/blob/main/src/client/scripts/components/Navbar1.js